README.md

```diff
@@ -383,3 +383,82 @@ For example, if a tagged VPC has a user-created (non-default) subnet that is not
+
+## Deploying as a Kubernetes CronJob
+
+Kubernetes [CronJobs](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/) schedule programs to run periodically or at a given point in time. Deploying grafiti to a Kubernetes allows you to clean your AWS account periodically, and aggregate and forward deletion logs. Creating and managing a grafiti CronJob can be made even easier using [Tectonic](https://coreos.com/tectonic/), CoreOS' self-driving Kubernetes software.
+
+### Setting up a CronJob
+
+A CronJob can be configured using a Kubernetes configuration file:
+```yaml
+apiVersion: batch/v2alpha1
+kind: CronJob
+metadata:
+  name: grafiti-deleter
+spec:
+  schedule: "* */6 * * *" # Run every 6 hours
+  jobTemplate:
+    spec:
+      template:
+        spec:
+          containers:
+          - command:
+            - /bin/bash
+            - -c
+            - grafiti -e -c /opt/config.toml delete -s --all-deps -f /opt/tags.json
+            env:
+              # Specify GRF_* environment variables here
+              - name: AWS_REGION
+                value: us-east-1
+              - name: AWS_CREDENTIAL_PROFILES_FILE # Sets the AWS 'credentials' file path
+                value: /etc/credentials
+            name: grafiti-deleter
+            image: your/registry/grafiti:v0.1.1
+            volumeMounts:
+              # Alternatively, add your own 'secret':
+              # https://kubernetes.io/docs/concepts/configuration/secret/
+              - mountPath: /etc/credentials # Mount a set of AWS credentials
+                name: grafiti-aws-credentials
+                readOnly: true
+              - mountPath: /opt/config.toml
+                name: config-path
+              - mountPath: /opt/tags.json
+                name: tags-path
+            securityContext:
+              runAsNonRoot: true
+              runAsUser: 1000
+          volumes:
+            - hostPath:
+                path: ~/.aws/credentials # Specify location of AWS credentials you want to mount
+              name: grafiti-aws-credentials
+            - hostPath:
+                path: ./config.toml # Add your own config file path here
+              name: config-path
+            - hostPath:
+                path: ./example-tags-input.json # Add your own tag file path here
+              name: tags-path
+          restartPolicy: OnFailure
+```
+
+1. Ensure container environments are provisioned with the following:
+    * Valid AWS credentials (environment variables or a 'credentials' file)
+    * A grafiti configuration file and/or environment variables
+    * Data or tag input files, depending on which sub-command you are running
+
+2. Run your Kubernetes API server with the `--runtime-config=batch/v2alpha1=true` flag to enable the CronJob API version. If you're using Tectonic, navigate to Console -> Workloads -> Daemon Sets -> YAML tab, and add a `- --runtime-config=batch/v2alpha1=true` field under the `containers.name:kube-apiserver.command` section.
```

Comment:
What happens to this config after a (tectonic/kubernetes) update? Does it disappear?